src/aot/CMakeLists.txt

```diff
@@ -2,3 +2,26 @@ add_library(aot aot.cpp)
 target_include_directories(aot PUBLIC ${CMAKE_SOURCE_DIR}/src)
 target_include_directories(aot PUBLIC ${CMAKE_BINARY_DIR})
 target_compile_definitions(aot PRIVATE ${BPFTRACE_FLAGS})
+
+# TODO(dxu): remove LLVM requirement
+# Currently the only LLVM API the runtime uses is llvm::itaniumDemangle()
+find_package(LLVM REQUIRED CONFIG)
```

Comment:
Hi @danobi ,

These aot runtime tests fails, when the bpftrace-aotrt is linked against libbcc. All aot tests fails in fedora34.

Reply:
LLVM ERROR: inconsistency in registered CommandLine options

Reply:
May be I have to set LIBBCC_BPF_CONTAINS_RUNTIME to 1 ?

Reply:
Try `-DENABLE_LLVM_NATIVECODEGEN=0` when compiling bcc

Reply:
Doesnt work with bcc v0.20.0. (Version which is present in fedora34).

Reply:
With latest bcc version, it seems that  ./src/aot/bpftrace-aotrt is linked against libbcc_bpf instead of libbcc. That's why it might work in that case.